docs/content/getting-started/cue.md

```diff
@@ -11,30 +11,30 @@ weight: 5
 CUE is a language specifically designed for configuration and is an integral component of the `hof` developer experience.
 {{</lead>}}
 
-The choice to use CUE was driven by two key considerations:
+Two key considerations drove the choice to use CUE:
 
 1. __Declarative Code Generation__<br>
-   `hof` uses code generators defined in CUE which combine data and templates to generate any file.
-   With CUE as the source of truth, we gain consistency for inputs and mappings allowing
-	 for composable generators to generate code across languages and technologies.
-	 `hof` allows you to define the declarative schemas to code generators and package them into reusable modules.
+   `hof` uses code generators defined in CUE, combining data and templates to generate files.
+   With CUE as the source of truth, we gain consistency for inputs and mappings, allowing
+    composable generators to generate code across languages and technologies.
+    `hof` allows you to define the declarative schemas to code generators and package them into reusable modules.
 
 2. __Purpose-Built for Large-Scale Configuration__<br>
-   CUE has a strong foundation and provides robust tooling for managing configuration across multiple files, packages, and modules.
+   CUE has a strong foundation and robust tooling for managing configuration across multiple files, packages, and modules.
    CUE's logical nature ensures that the configuration is both efficient and correct.
-	 Since `hof`'s inputs are typically declarative configuration, CUE is a natural fit throughout the process.
+    Since `hof`'s inputs are typically declarative configurations, CUE is a natural fit throughout the process.
 
 Other benefits of CUE include:
 
-- Configuration-like syntax with a full, logical language.
-- Built on sound principles and a rich heritage.
+- Configuration-like syntax with a whole, logical language.
```

Comment:
not sure I like "whole" here, I haven't heard that word used with a programming language

maybe we make it "... with a proper language, including ... something ..."?

What do you think

Reply:
```suggestion
- Configuration syntax with a complete, logical language.
```

Reply:
what do you think of this instead?

Reply:
I like the `-like` because it isn't exactly configuration syntax, but then maybe it's not worth pointing that out here...

Let me think on it overnight

Reply:
other suggestions:

- Configuration-style syntax with a comprehensive, logical programming language.
- A logical, full-featured programming language using a configuration-like syntax.
- Configuration-like syntax that allows for full logical programming language features.

Reply:
Actually, looking at the later bullets they contain the same information we are debating

what about something like "Configuration-style syntax that is familiar, yet fresh"